emgapianns/management/commands/import_summary.py

```diff
@@ -14,20 +27,16 @@
 
 class Command(EMGBaseCommand):
 
-    FUNCTION_SUB_DIR = 'functional-annotation'
-    PATHWAY_SUB_DIR = 'pathways-systems'
-
     def __init__(self):
         super().__init__()
-        self.suffixes = ['.ips', '.ipr', '.go', '.go_slim', '.pfam', '.ko', '.gprops', '.antismash']
-        self.pathway_suffixes = ['.gprops', '.antismash', '.kegg_pathways']
-        self.kegg_pathway_suffix = ['.kegg_pathways']
+        self.suffixes = [".ipr", ".go", ".go_slim", ".pfam", ".ko", ".paths.gprops", ".antismash"]
```

Comment:
The format of the InterProScan summary file change completely between version 4 and 5. ipr will support v4 and ips v5 functionality. I cannot see that functionality anymore?